transifex/utils.py

```diff
@@ -297,6 +297,17 @@ def merge_pr(self):
                         'Failed to merge [%s/#%d], because of the exception [%s]',
                         self.name, self.pr.number, e,
                     )
+                    try:
+                        # If it's possible, a PR message would be appreciated in these cases.
+                        # Github may just be super borked though, hence the empty catch.
+                        self.pr.create_issue_comment(
+                            '@{owner} a github exception prevented this PR from being automatically merged:\n`{error}`'.format(
+                                owner=self.owner,
+                                error=e
+                            )
+                        )
+                    except:
+                        pass
```

Comment:
A GithubException could occur because Github is down. In that case, leaving a PR comment would also fail. Furthermore, I'm not sure that opening the PR makes sense at all here since we know that it will always fail. I also think that the spartan team recently added logic to automatically remove the PR/branch from Github when the script cleans up, so it may disappear before you have a chance to merge it.